tests/pipeline/test_protonet.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+import pytorch_lightning as pl
+import torch
+from torch.utils.data import DataLoader, TensorDataset
+from yacs.config import CfgNode as CfgNode
+
+from kale.embed.image_cnn import ResNet18Feature
+from kale.pipeline.protonet import ProtoNetTrainer
+
+url = "https://github.com/pykale/data/raw/main/images/omniglot/omniglot_demo.zip"
```

Comment:
Isn't it defined already in `download_test_data`?

Reply:
Deleted.